src/java/META-INF/beans.xml

```diff
@@ -0,0 +1,20 @@
+<beans xmlns="https://jakarta.ee/xml/ns/jakartaee" 
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+       xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/beans_4_0.xsd" 
+       version="4.0" bean-discovery-mode="annotated">
+       
+       <alternatives>
```

Comment:
Please remove the commented tags. The commented tags  are related to lutece-core.